jans-linux-setup/jans_setup/templates/jans-saml/clients.json

```diff
@@ -21,7 +21,7 @@
     "description": "Keycloak OpenID client used for SAML authentication",
     "scopes_dns": [],
     "scopes_ids": ["email" , "user_name" , "openid"],
-    "redirect_uri": ["https://%(idp_config_hostname)s:%(idp_config_http_port)s/realms/master/kc-jans-authn-rest-bridge/auth-complete"],
+    "redirect_uri": ["http://%(idp_config_hostname)s:%(idp_config_http_port)s/realms/master/kc-jans-authn-rest-bridge/auth-complete"],
```

Comment:
@devrimyatar @uprightech There could be inconsistency here. The realm in use is `master`, but we created execution config for this client under `jans-api` realm. See https://github.com/JanssenProject/jans/blob/main/jans-linux-setup/jans_setup/setup_app/installers/jans_saml.py#L250 and https://github.com/JanssenProject/jans/blob/main/jans-linux-setup/jans_setup/templates/jans-saml/kc_jans_api/jans.execution-config-jans.json#L5